docker-compose.yml

```diff
@@ -19,6 +19,11 @@ services:
     image: redis
     ports:
     - "6378:6379"
+  imagor:
+    image: shumc/imagorvideo
+    ports:
+    - "8005:8000"
+    command: ["-imagor-unsafe"]
```

Comment:
Does unsafe relate to no https or something else?

Reply:
refers to not needing a signing key